Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Matomo/Plugin LoginLdap](https://hosted.weblate.org/projects/matomo/plugin-loginldap/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/matomo/plugin-loginldap/horizontal-auto.svg)

snake14 commented on lang/it.json:
```diff
@@ -36,7 +36,7 @@
         "LdapUserAccessAttributeServerSeparatorDescription": "Stringa usata per separare gli ID delle istanze server dagli ID elenchi siti. Se impostata a ':', ci si aspetterà un attributo di accesso simile a '<server-id>:<site-list>;<server-id>:<site-list>'.",
         "LdapUserAccessAttributeServerSpecDelimiter": "Delimitatore Specifiche Server Attributi Accesso Utente",
-        "LdapUserAccessAttributeServerSpecDelimiterDescription": "Stringa utilizzata per delimitare le specifiche del server in un attributo di accesso utente. Se impostata a ';', ci si aspetta che l'attributo di accesso sia simile a '<server-spec>;<server-spec>;...'.",
+        "LdapUserAccessAttributeServerSpecDelimiterDescription": "Stringa utilizzata per delimitare le specifiche del server in un attributo di accesso utente. Se impostata a ';', ci si aspetta che l'attributo di accesso sia simile a '<server-spec>;<server-spec>;…'.</server-spec></server-spec>",
```
@sgiehl Should I still merge this even though it's adding some HTML that I don't think is supposed to be there?

sgiehl replied:
If you see obvious mistakes might be good to fix them on weblate, so they get fixed before merging.

snake14 replied:
@sgiehl I don't think I currently have a Weblate account. Can I just create one?

sgiehl replied:
Yes. Sure. Send me the nickname and I can also give you higher permissions.

snake14 replied:
@sgiehl I created a Weblate user `snake14`. I think I made the change, but I'm not sure if it went through.

snake14 replied:
@sgiehl It looks like it didn't work because it's Weblate that's making the HTML change. Note the notification at the top:
![image](https://github.com/matomo-org/plugin-LoginLdap/assets/11394099/e48aec84-6fc6-4016-94bb-0443997afd62)

AltamashShaikh replied:
@sgiehl Any idea what issue @snake14 is facing ?

sgiehl replied:
This should be fixed now. We can't use text that looks like html tags unless they are html tags. Weblate automatically closes open html tags at the end of a string. I've changed the English original text to use html entities (`&gt;` / `&lt;`) instead and also updated all translations to do so.

snake14 replied:
Thank you @sgiehl . I'll watch out for that in the future.

weblate committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (93 of 93 strings)

Translation: Matomo/Plugin LoginLdap
Translate-URL: https://hosted.weblate.org/projects/matomo/plugin-loginldap/pt_BR/

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Stefan <stefan@matomo.org>

weblate committed:
Translated using Weblate (Gujarati)

Currently translated at 100.0% (93 of 93 strings)

Translation: Matomo/Plugin LoginLdap
Translate-URL: https://hosted.weblate.org/projects/matomo/plugin-loginldap/gu/

[ci skip]

Translated using Weblate (Gujarati)

Currently translated at 100.0% (93 of 93 strings)

Translation: Matomo/Plugin LoginLdap
Translate-URL: https://hosted.weblate.org/projects/matomo/plugin-loginldap/gu/

[ci skip]

Translated using Weblate (Gujarati)

Currently translated at 100.0% (93 of 93 strings)

Translation: Matomo/Plugin LoginLdap
Translate-URL: https://hosted.weblate.org/projects/matomo/plugin-loginldap/gu/

[ci skip]

Translated using Weblate (Gujarati)

Currently translated at 64.5% (60 of 93 strings)

Translation: Matomo/Plugin LoginLdap
Translate-URL: https://hosted.weblate.org/projects/matomo/plugin-loginldap/gu/

[ci skip]

Translated using Weblate (Gujarati)

Currently translated at 32.2% (30 of 93 strings)

Translation: Matomo/Plugin LoginLdap
Translate-URL: https://hosted.weblate.org/projects/matomo/plugin-loginldap/gu/

[ci skip]

Translated using Weblate (Gujarati)

Currently translated at 27.9% (26 of 93 strings)

Translation: Matomo/Plugin LoginLdap
Translate-URL: https://hosted.weblate.org/projects/matomo/plugin-loginldap/gu/

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Kalpesh Mahida <kalpesh.mahida@gmail.com>
Co-authored-by: Stefan <stefan@matomo.org>

weblate committed:
Translated using Weblate (German)

Currently translated at 100.0% (93 of 93 strings)

Translation: Matomo/Plugin LoginLdap
Translate-URL: https://hosted.weblate.org/projects/matomo/plugin-loginldap/de/

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Stefan <stefan@matomo.org>

weblate committed:
Translated using Weblate (Turkish)

Currently translated at 100.0% (93 of 93 strings)

Translation: Matomo/Plugin LoginLdap
Translate-URL: https://hosted.weblate.org/projects/matomo/plugin-loginldap/tr/

[ci skip]

Translated using Weblate (Turkish)

Currently translated at 100.0% (93 of 93 strings)

Translation: Matomo/Plugin LoginLdap
Translate-URL: https://hosted.weblate.org/projects/matomo/plugin-loginldap/tr/

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Kaya Zeren <kayazeren@gmail.com>
Co-authored-by: Stefan <stefan@matomo.org>

weblate committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (93 of 93 strings)

Translation: Matomo/Plugin LoginLdap
Translate-URL: https://hosted.weblate.org/projects/matomo/plugin-loginldap/ca/

[ci skip]

Translated using Weblate (Catalan)

Currently translated at 100.0% (93 of 93 strings)

Translation: Matomo/Plugin LoginLdap
Translate-URL: https://hosted.weblate.org/projects/matomo/plugin-loginldap/ca/

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Stefan <stefan@matomo.org>
Co-authored-by: raf <rafroset@gmail.com>

weblate committed:
Translated using Weblate (Japanese)

Currently translated at 100.0% (93 of 93 strings)

Translation: Matomo/Plugin LoginLdap
Translate-URL: https://hosted.weblate.org/projects/matomo/plugin-loginldap/ja/

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Stefan <stefan@matomo.org>

weblate committed:
Translated using Weblate (Danish)

Currently translated at 86.0% (80 of 93 strings)

Translation: Matomo/Plugin LoginLdap
Translate-URL: https://hosted.weblate.org/projects/matomo/plugin-loginldap/da/

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Stefan <stefan@matomo.org>

weblate committed:
Translated using Weblate (Greek)

Currently translated at 100.0% (93 of 93 strings)

Translation: Matomo/Plugin LoginLdap
Translate-URL: https://hosted.weblate.org/projects/matomo/plugin-loginldap/el/

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Stefan <stefan@matomo.org>

weblate committed:
Translated using Weblate (Polish)

Currently translated at 92.4% (86 of 93 strings)

Translation: Matomo/Plugin LoginLdap
Translate-URL: https://hosted.weblate.org/projects/matomo/plugin-loginldap/pl/

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Stefan <stefan@matomo.org>

weblate committed:
Translated using Weblate (Czech)

Currently translated at 92.4% (86 of 93 strings)

Translation: Matomo/Plugin LoginLdap
Translate-URL: https://hosted.weblate.org/projects/matomo/plugin-loginldap/cs/

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Stefan <stefan@matomo.org>

weblate committed:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (93 of 93 strings)

Translation: Matomo/Plugin LoginLdap
Translate-URL: https://hosted.weblate.org/projects/matomo/plugin-loginldap/pt/

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Stefan <stefan@matomo.org>

weblate committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (93 of 93 strings)

Translation: Matomo/Plugin LoginLdap
Translate-URL: https://hosted.weblate.org/projects/matomo/plugin-loginldap/it/

[ci skip]

Translated using Weblate (Italian)

Currently translated at 98.9% (92 of 93 strings)

Translation: Matomo/Plugin LoginLdap
Translate-URL: https://hosted.weblate.org/projects/matomo/plugin-loginldap/it/

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Stefan <stefan@matomo.org>
Co-authored-by: Vincenzo Ferrara <v.ferrara58@gmail.com>

weblate committed:
Translated using Weblate (Dutch)

Currently translated at 100.0% (93 of 93 strings)

Translation: Matomo/Plugin LoginLdap
Translate-URL: https://hosted.weblate.org/projects/matomo/plugin-loginldap/nl/

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Stefan <stefan@matomo.org>

weblate committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (93 of 93 strings)

Translation: Matomo/Plugin LoginLdap
Translate-URL: https://hosted.weblate.org/projects/matomo/plugin-loginldap/es/

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Stefan <stefan@matomo.org>

weblate committed:
Translated using Weblate (French)

Currently translated at 95.6% (89 of 93 strings)

Translation: Matomo/Plugin LoginLdap
Translate-URL: https://hosted.weblate.org/projects/matomo/plugin-loginldap/fr/

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Stefan <stefan@matomo.org>

weblate committed:
Translated using Weblate (Albanian)

Currently translated at 100.0% (93 of 93 strings)

Translation: Matomo/Plugin LoginLdap
Translate-URL: https://hosted.weblate.org/projects/matomo/plugin-loginldap/sq/

[ci skip]

Translated using Weblate (Albanian)

Currently translated at 100.0% (93 of 93 strings)

Translation: Matomo/Plugin LoginLdap
Translate-URL: https://hosted.weblate.org/projects/matomo/plugin-loginldap/sq/

[ci skip]

Co-authored-by: Besnik Bleta <besnik@programeshqip.org>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Stefan <stefan@matomo.org>

weblate committed:
Translated using Weblate (Swedish)

Currently translated at 100.0% (93 of 93 strings)

Translation: Matomo/Plugin LoginLdap
Translate-URL: https://hosted.weblate.org/projects/matomo/plugin-loginldap/sv/

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Stefan <stefan@matomo.org>

weblate committed:
Translated using Weblate (Serbian)

Currently translated at 92.4% (86 of 93 strings)

Translation: Matomo/Plugin LoginLdap
Translate-URL: https://hosted.weblate.org/projects/matomo/plugin-loginldap/sr/

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Stefan <stefan@matomo.org>

weblate committed:
Translated using Weblate (Chinese (Traditional))

Currently translated at 92.4% (86 of 93 strings)

Translation: Matomo/Plugin LoginLdap
Translate-URL: https://hosted.weblate.org/projects/matomo/plugin-loginldap/zh_Hant/

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Stefan <stefan@matomo.org>

weblate committed:
Translated using Weblate (Norwegian Bokmål)

Currently translated at 68.8% (64 of 93 strings)

Translation: Matomo/Plugin LoginLdap
Translate-URL: https://hosted.weblate.org/projects/matomo/plugin-loginldap/nb_NO/

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Stefan <stefan@matomo.org>

weblate committed:
Translated using Weblate (Spanish (Argentina))

Currently translated at 94.6% (88 of 93 strings)

Translation: Matomo/Plugin LoginLdap
Translate-URL: https://hosted.weblate.org/projects/matomo/plugin-loginldap/es_AR/

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Stefan <stefan@matomo.org>

weblate committed:
Translated using Weblate (Chinese (Simplified))

Currently translated at 92.4% (86 of 93 strings)

Translation: Matomo/Plugin LoginLdap
Translate-URL: https://hosted.weblate.org/projects/matomo/plugin-loginldap/zh_Hans/

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Stefan <stefan@matomo.org>
